lib/components/ChartContainer/ChartContainer.css

```diff
@@ -2,6 +2,7 @@
   border-radius: 5px;
   border: 1px solid var(--vds-color-slate-darker);
   color: var(--vds-color-slate-darkest);
+  width: 984px;
```

Comment:
maybe it should be `min-width`?

Reply:
oh i was thinking more like `max-width` since it's the same width in all the mocks, but up to you since you're the one using it?